flake.lock

```diff
@@ -0,0 +1,82 @@
+{
+  "nodes": {
+    "flake-utils": {
+      "locked": {
+        "lastModified": 1620759905,
+        "narHash": "sha256-WiyWawrgmyN0EdmiHyG2V+fqReiVi8bM9cRdMaKQOFg=",
+        "owner": "numtide",
+        "repo": "flake-utils",
+        "rev": "b543720b25df6ffdfcf9227afafc5b8c1fabfae8",
+        "type": "github"
+      },
+      "original": {
+        "owner": "numtide",
+        "repo": "flake-utils",
+        "type": "github"
+      }
+    },
+    "idris": {
+      "inputs": {
+        "flake-utils": [
+          "flake-utils"
+        ],
+        "idris-emacs-src": "idris-emacs-src",
+        "nixpkgs": [
+          "nixpkgs"
+        ]
+      },
+      "locked": {
+        "lastModified": 1621286848,
+        "narHash": "sha256-ZQaRlsVYj9JL2ZXqViQO4uU8qmOP9e28uhzq0+vwwrU=",
+        "owner": "idris-lang",
+        "repo": "Idris2",
+        "rev": "2f66f3e006bbbe73605db30cdb8c2464d41bb49c",
+        "type": "github"
+      },
+      "original": {
+        "owner": "idris-lang",
+        "repo": "Idris2",
+        "type": "github"
+      }
+    },
+    "idris-emacs-src": {
```

Comment:
Is `idris-emacs-src` required? (i'm totally unfamiliar with flake, i guess it's generated, but may be this dependency comes from your environment and is not needed?)

Reply:
I am not sure, the flake itself was generated from the official Idris2 flake (flakes can generate templates too). I think the emacs dependency is there because there must be a target that installs emacs along with REPLica but I dont know the name of the target, I've just used the default one. I use neovim but if you are interested I can look into it. This adds potentially unused dependencies but as it was generated , I would prefer to minimize the changes, and eventually notify upstream about it,

Reply:
It's a really minor issue, I think we can ship it as-is and we'll try to figure it out later if needed.